index.js

```diff
@@ -137,13 +170,26 @@ const setWebhook = async (webhookUrl, auth, env) => {
   }
 
   const response = await post(requestConfig);
-  const body = JSON.parse(response.body);
 
-  if (body.errors) {
-    throw new WebhookURIError(response.body);
-    return null;
+  switch (response.statusCode) {
+    case 200:
+    case 204:
+      break;
+    case 403:
+     throw new WebhookURIError(response);
+     return null;
+    case 429:
+      throw new RateLimimitError(response);
```

Comment:
Not to sound like a broken record or come off as critical but marking another typo here just in case